bindata/manifests/daemon/daemonset.yaml

```diff
@@ -92,6 +92,23 @@ spec:
               mountPath: /host/etc/os-release
               readOnly: true
         {{- end }}
+        {{- if ne .RDMACNIImage "" }}
```

Comment:
nit: please can the if statement be the same as `{{- if .OVSCNIImage }}`

Reply:
done